src/Resources/contao/languages/sr/tl_settings.xlf

```diff
@@ -126,7 +126,7 @@
         <target>Учини IP адресе анонимним</target>
       </trans-unit>
       <trans-unit id="tl_settings.privacyAnonymizeIp.1">
-        <source>Anonymize any IP address that is stored in the database, except in the &lt;em&gt;tl_session&lt;/em&gt; table (the IP address is bound to the session for security reasons).</source>
+        <source>Anonymize any IP address that is stored in the database.</source>
```

Comment:
You have to change `en/tl_settings.xlf` not `sr/tl_settings.xlf`. 😄

Reply:
Fixed in d518a871149eb38d5d66fec432d25ea6869ddcaf